jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/clouddebug/nodejs/NodeJsDebuggerSupport.kt

```diff
@@ -24,6 +24,7 @@ class NodeJsDebuggerSupport : DebuggerSupport() {
     override val debuggerPath = null
 
     private val NODE_EXECUTABLE = "node"
+    private val NODE_EXECUTABLE2 = "nodejs"
```

Comment:
Would it make sense to put these in a `NODE_EXECUTABLES`  set? Then we could just do exename not in ....